# Assignment 05

**Wale Agboola**

1)Explore College Scorecard website - https://collegescorecard.ed.gov/

2)Explore College Scorecard Data web page - https://collegescorecard.ed.gov/data/

3)Download and read the documentation -   https://collegescorecard.ed.gov/assets/FullDataDocumentation.pdf

4)Download the dataset - https://ed-public-download.app.cloud.gov/downloads/CollegeScorecard_Raw_Data.zip (265MB Zip File)

5)Unzip the file into a folder in your personal computer (there are many files, one for each school year)

**6)Upload the 2017-2018 school year file "MERGED2017_18_PP.csv" to Colab/Deepnote or any Cloud environment of your choice (you can use your local environment too) and create a Jupyter notebook to explore the dataset:**

In [2]:
import pandas as pd

df = pd.read_csv('MERGED2017_18_PP.csv')


**how many colleges or rows?**

In [3]:
rows = len(df.axes[0]) 

print(rows)

7112


There are 7,112 colleges (or rows) in the dataset.

**how many variables or columns?**

In [4]:
cols = len(df.axes[1]) 

print(cols)

1986


There are 1,986 columns in the dataset.

**Look at the first 5 colleges**

In [5]:
df.head()

UNITID    OPEID  OPEID6                               INSTNM        CITY  \
0  100654   100200    1002             Alabama A & M University      Normal   
1  100663   105200    1052  University of Alabama at Birmingham  Birmingham   
2  100690  2503400   25034                   Amridge University  Montgomery   
3  100706   105500    1055  University of Alabama in Huntsville  Huntsville   
4  100724   100500    1005             Alabama State University  Montgomery   

  STABBR         ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
0     AL       35762           NaN      NaN     NaN  ...   
1     AL  35294-0110           NaN      NaN     NaN  ...   
2     AL  36117-3553           NaN      NaN     NaN  ...   
3     AL       35899           NaN      NaN     NaN  ...   
4     AL  36104-0271           NaN      NaN     NaN  ...   

   OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN   G12MN  \
0                           NaN       0.7106        0.7418   5207.0  1185.0   
1                           NaN       0.4009        0.5488  13308.0  9888.0   
2                           NaN       0.5000        0.5000    431.0   437.0   
3                           NaN       0.2012        0.4231   7519.0  2378.0   
4                           NaN       0.7594        0.7402   5229.0   687.0   

   SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
0   1410.0                 NaN                      NaN   
1   1948.0                 NaN                      NaN   
2      4.0                 NaN                      NaN   
3   1203.0                 NaN                      NaN   
4   1143.0                 NaN                      NaN   

   FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
0                       NaN             NaN  
1                       NaN             NaN  
2                       NaN             NaN  
3                       NaN             NaN  
4                       NaN             NaN  

[5 rows x 1986 columns]

**look at the last 5 colleges**

In [6]:
df.tail()

UNITID    OPEID  OPEID6  \
7107  48312404   108163    1081   
7108  48387801  4223701   42237   
7109  48400201  4228101   42281   
7110  48778201  4221501   42215   
7111  49005401  4182601   41826   

                                                 INSTNM             CITY  \
7107                   Arizona State University at Yuma             Yuma   
7108  Bay Area Medical Academy - San Jose Satellite ...         San Jose   
7109           High Desert Medical College - Bakerfield      Bakersfield   
7110                                       BCI - Malden           Malden   
7111               HCI College - Fort Lauderdale Campus  Fort Lauderdale   

     STABBR        ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
7107     AZ  853656900           NaN      NaN     NaN  ...   
7108     CA      95113           NaN      NaN     NaN  ...   
7109     CA      93301           NaN      NaN     NaN  ...   
7110     MA  021480000           NaN      NaN     NaN  ...   
7111     FL      33309           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN  UG12MN  G12MN  \
7107                           NaN          NaN           NaN     NaN    NaN   
7108                           NaN          NaN           NaN     NaN    NaN   
7109                           NaN          NaN           NaN     NaN    NaN   
7110                           NaN          NaN           NaN     NaN    NaN   
7111                           NaN          NaN           NaN     NaN    NaN   

      SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
7107      NaN                 NaN                      NaN   
7108      NaN                 NaN                      NaN   
7109      NaN                 NaN                      NaN   
7110      NaN                 NaN                      NaN   
7111      NaN                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
7107                       NaN             NaN  
7108                       NaN             NaN  
7109                       NaN             NaN  
7110                       NaN             NaN  
7111                       NaN             NaN  

[5 rows x 1986 columns]

**Look at a random sample of 5 colleges**

In [7]:
df.sample(5)

UNITID    OPEID  OPEID6                                  INSTNM  \
3246    218663   344800    3448   University of South Carolina-Columbia   
4680    430245  3035303   30353  Southern Careers Institute-San Antonio   
6931  23368402   145906    1459    Strayer University-Woodbridge Campus   
1859    176451  2516200   25162                Wesley Biblical Seminary   
3799    237473   835500    8355        James Rumsey Technical Institute   

             CITY STABBR         ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
3246     Columbia     SC       29208           NaN      NaN     NaN  ...   
4680  San Antonio     TX       78221           NaN      NaN     NaN  ...   
6931   Woodbridge     VA       22192           NaN      NaN     NaN  ...   
1859    Ridgeland     MS  39157-1920           NaN      NaN     NaN  ...   
3799  Martinsburg     WV  25403-0259           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN  \
3246                           NaN       0.1544        0.4582  26791.0   
4680                           NaN       0.6583        0.6278    926.0   
6931                           NaN          NaN           NaN      NaN   
1859                           NaN          NaN           NaN      NaN   
3799                           NaN       0.4844        0.0000    133.0   

        G12MN  SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
3246  10022.0   5105.0                 NaN                      NaN   
4680      NaN    720.0                 NaN                      NaN   
6931      NaN      NaN                 NaN                      NaN   
1859    107.0      NaN                 NaN                      NaN   
3799      NaN    128.0                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
3246                       NaN             NaN  
4680                       NaN             NaN  
6931                       NaN             NaN  
1859                       NaN             NaN  
3799                       NaN             NaN  

[5 rows x 1986 columns]

**7)Since there are too many variables to explore, many of which are not interesting. Create a smaller data frame with the following 9 variables:
UNITID,
INSTNM,
CITY,
STABBR,
ZIP,
ADM_RATE,
UGDS,
TUITIONFEE_IN,
MN_EARN_WNE_P10**

In [8]:
small_df = df[['UNITID', 'INSTNM', 'CITY', 'STABBR', 'ZIP', 'ADM_RATE', 'UGDS', 
               'TUITIONFEE_IN', 'MN_EARN_WNE_P10']]

small_df.head()

UNITID                               INSTNM        CITY STABBR         ZIP  \
0  100654             Alabama A & M University      Normal     AL       35762   
1  100663  University of Alabama at Birmingham  Birmingham     AL  35294-0110   
2  100690                   Amridge University  Montgomery     AL  36117-3553   
3  100706  University of Alabama in Huntsville  Huntsville     AL       35899   
4  100724             Alabama State University  Montgomery     AL  36104-0271   

   ADM_RATE     UGDS  TUITIONFEE_IN  MN_EARN_WNE_P10  
0    0.9027   4824.0         9857.0              NaN  
1    0.9181  12866.0         8328.0              NaN  
2       NaN    318.0         6900.0              NaN  
3    0.8123   6917.0        10280.0              NaN  
4    0.9787   4189.0        11068.0              NaN

**8)Download and read (use naked eyes, no coding) the data dictionary file - https://collegescorecard.ed.gov/assets/CollegeScorecardDataDictionary.xlsx**

**9)Find out the definition of the above 9 variables and document the definitions in the Jupyter Notebook in a Markdown cell so that the readers of your notebook will know the meaning of these variables.**

UNITID - Unit ID for Institution

INSTNM - Institution Name

CITY - City Institution is located in

STABBR - State Institution is located in

ZIP - Zip Code of Institution

ADM_RATE - Admission Rate

UGDS - Enrollment of undergraduate certificate/degree-seeking students

TUITIONFEE_IN - In-state tuition and fees

MN_EARN_WNE_P10 - Mean earnings of students working and not enrolled 10 years after entry

**10)Explore the smaller data frame**

**Look at the first 10 colleges**

In [9]:
small_df.head(10)

UNITID                               INSTNM            CITY STABBR  \
0  100654             Alabama A & M University          Normal     AL   
1  100663  University of Alabama at Birmingham      Birmingham     AL   
2  100690                   Amridge University      Montgomery     AL   
3  100706  University of Alabama in Huntsville      Huntsville     AL   
4  100724             Alabama State University      Montgomery     AL   
5  100751            The University of Alabama      Tuscaloosa     AL   
6  100760    Central Alabama Community College  Alexander City     AL   
7  100812              Athens State University          Athens     AL   
8  100830      Auburn University at Montgomery      Montgomery     AL   
9  100858                    Auburn University          Auburn     AL   

          ZIP  ADM_RATE     UGDS  TUITIONFEE_IN  MN_EARN_WNE_P10  
0       35762    0.9027   4824.0         9857.0              NaN  
1  35294-0110    0.9181  12866.0         8328.0              NaN  
2  36117-3553       NaN    318.0         6900.0              NaN  
3       35899    0.8123   6917.0        10280.0              NaN  
4  36104-0271    0.9787   4189.0        11068.0              NaN  
5  35487-0100    0.5330  32387.0        10780.0              NaN  
6       35010       NaN   1404.0         4440.0              NaN  
7       35611       NaN   2801.0            NaN              NaN  
8  36117-3596    0.8240   4211.0         8020.0              NaN  
9       36849    0.8393  23391.0        10968.0              NaN

**look at the last 10 colleges**

In [10]:
small_df.tail(10)

UNITID                                             INSTNM  \
7102  48154401  National Personal Training Institute of Cleveland   
7103  48305401             Barber School of Pittsburgh - Ambridge   
7104  48312401                 Arizona State University at Tucson   
7105  48312402  Arizona State University - ASU Colleges at Lak...   
7106  48312403        Arizona State University at The Gila Valley   
7107  48312404                   Arizona State University at Yuma   
7108  48387801  Bay Area Medical Academy - San Jose Satellite ...   
7109  48400201           High Desert Medical College - Bakerfield   
7110  48778201                                       BCI - Malden   
7111  49005401               HCI College - Fort Lauderdale Campus   

                  CITY STABBR        ZIP  ADM_RATE  UGDS  TUITIONFEE_IN  \
7102  Highland Heights     OH      44143       NaN   NaN            NaN   
7103          Ambridge     PA  150030000       NaN   NaN            NaN   
7104            Tucson     AZ  857454284       NaN   NaN            NaN   
7105  Lake Havasu City     AZ  864036877       NaN   NaN         6904.0   
7106          Thatcher     AZ  855525545       NaN   NaN            NaN   
7107              Yuma     AZ  853656900       NaN   NaN            NaN   
7108          San Jose     CA      95113       NaN   NaN            NaN   
7109       Bakersfield     CA      93301       NaN   NaN            NaN   
7110            Malden     MA  021480000       NaN   NaN            NaN   
7111   Fort Lauderdale     FL      33309       NaN   NaN        22575.0   

      MN_EARN_WNE_P10  
7102              NaN  
7103              NaN  
7104              NaN  
7105              NaN  
7106              NaN  
7107              NaN  
7108              NaN  
7109              NaN  
7110              NaN  
7111              NaN

**Look at a random sample of 10 colleges**

In [11]:
small_df.sample(10)

UNITID                                             INSTNM         CITY  \
3809  237686            West Virginia University at Parkersburg  Parkersburg   
4706  431275    Herkimer County BOCES-Practical Nursing Program        Ilion   
4609  420130                 Prism Career Institute-Cherry Hill  Cherry Hill   
4208  372073                                    Fremont College     Cerritos   
4347  384342                        Heartland Community College       Normal   
5333  450599                      Empire Beauty School-Richmond     Richmond   
2776  205090  Firelands Regional Medical Center School of Nu...     Sandusky   
4476  412544      Shear Ego International School of Hair Design    Rochester   
164   107600                            Philander Smith College  Little Rock   
4314  382063                  University of Phoenix-Puerto Rico     Guaynabo   

     STABBR         ZIP  ADM_RATE    UGDS  TUITIONFEE_IN  MN_EARN_WNE_P10  
3809     WV       26104       NaN  2172.0         3552.0              NaN  
4706     NY       13357       NaN    97.0            NaN              NaN  
4609     NJ       08002       NaN    69.0            NaN              NaN  
4208     CA       90703       NaN   151.0            NaN              NaN  
4347     IL  61761-9446       NaN  3677.0         4440.0              NaN  
5333     VA       23294       NaN    72.0            NaN              NaN  
2776     OH  44870-4788       NaN    49.0            NaN              NaN  
4476     NY       14617       NaN   107.0            NaN              NaN  
164      AR  72202-3799       NaN   887.0        12714.0              NaN  
4314     PR  00968-3028       NaN    44.0            NaN              NaN

**Find and display UMBC**

In [12]:
df[df['CITY']=='Baltimore']

UNITID    OPEID  OPEID6                                   INSTNM  \
1451    161855  2494700   24947          Baltimore Studio of Hair Design   
1452    161864   206100    2061         Baltimore City Community College   
1453    161873   210200    2102                  University of Baltimore   
1460    162283   206800    2068                  Coppin State University   
1462    162371  2545400   25454             North American Trade Schools   
1466    162654   207300    2073                          Goucher College   
1473    162928   207700    2077                 Johns Hopkins University   
1475    163046   207800    2078               Loyola University Maryland   
1477    163259   210400    2104         University of Maryland Baltimore   
1478    163268   210500    2105  University of Maryland-Baltimore County   
1480    163295   208000    2080        Maryland Institute College of Art   
1485    163453   208300    2083                  Morgan State University   
1487    163532   208700    2087            Ner Israel Rabbinical College   
1489    163578   206500    2065        Notre Dame of Maryland University   
1491    163736   749100    7491             Brightwood College-Baltimore   
1493    163815  2331000   23310         Maryland Beauty Academy of Essex   
3004    212452  3667300   36673               Faith Theological Seminary   
4225    373784  3493300   34933               All-State Career-Baltimore   
4762    434672   206300    2063    Community College of Baltimore County   
5314    450076  3493301   34933               Fortis Institute-Baltimore   
5834    461379  4168900   41689      Holistic Massage Training Institute   
6016    476601  4188400   41884  Bais HaMedrash and Mesivta of Baltimore   
7006  43018403   145924    1459    Strayer University-White Marsh Campus   

           CITY STABBR         ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
1451  Baltimore     MD  21201-3444           NaN      NaN     NaN  ...   
1452  Baltimore     MD  21215-7893           NaN      NaN     NaN  ...   
1453  Baltimore     MD  21201-5720           NaN      NaN     NaN  ...   
1460  Baltimore     MD  21216-3698           NaN      NaN     NaN  ...   
1462  Baltimore     MD       21244           NaN      NaN     NaN  ...   
1466  Baltimore     MD  21204-2794           NaN      NaN     NaN  ...   
1473  Baltimore     MD  21218-2688           NaN      NaN     NaN  ...   
1475  Baltimore     MD  21210-2699           NaN      NaN     NaN  ...   
1477  Baltimore     MD  21201-1627           NaN      NaN     NaN  ...   
1478  Baltimore     MD       21250           NaN      NaN     NaN  ...   
1480  Baltimore     MD  21217-4134           NaN      NaN     NaN  ...   
1485  Baltimore     MD  21251-0001           NaN      NaN     NaN  ...   
1487  Baltimore     MD       21208           NaN      NaN     NaN  ...   
1489  Baltimore     MD  21210-2476           NaN      NaN     NaN  ...   
1491  Baltimore     MD  21227-1063           NaN      NaN     NaN  ...   
1493  Baltimore     MD       21224           NaN      NaN     NaN  ...   
3004  Baltimore     MD  21212-2624           NaN      NaN     NaN  ...   
4225  Baltimore     MD       21224           NaN      NaN     NaN  ...   
4762  Baltimore     MD       21237           NaN      NaN     NaN  ...   
5314  Baltimore     MD       21244           NaN      NaN     NaN  ...   
5834  Baltimore     MD       21218           NaN      NaN     NaN  ...   
6016  Baltimore     MD  21209-1613           NaN      NaN     NaN  ...   
7006  Baltimore     MD       21236           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN  \
1451                           NaN       0.9375        0.9375     64.0   
1452                           NaN       0.7003        0.0000   5770.0   
1453                           NaN       0.5145        0.5000   3827.0   
1460                           NaN       0.6710        0.7232   2921.0   
1462                           NaN  

In [13]:
df[df['INSTNM']=='University of Maryland-Baltimore County']

UNITID   OPEID  OPEID6                                   INSTNM  \
1478  163268  210500    2105  University of Maryland-Baltimore County   

           CITY STABBR    ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
1478  Baltimore     MD  21250           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN  \
1478                           NaN       0.2207        0.4045  12702.0   

       G12MN  SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
1478  2998.0   1518.0                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
1478                       NaN             NaN  

[1 rows x 1986 columns]

**Find Univ. of Maryland College Park**

In [14]:
df[df['INSTNM']=='University of Maryland-College Park']

UNITID   OPEID  OPEID6                               INSTNM  \
1479  163286  210300    2103  University of Maryland-College Park   

              CITY STABBR    ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
1479  College Park     MD  20742           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN  \
1479                           NaN       0.1344        0.3126  32022.0   

        G12MN  SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
1479  11669.0   4568.0                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
1479                       NaN             NaN  

[1 rows x 1986 columns]

**Find and display Loyola College Maryland**

In [15]:
df[df['INSTNM']=='Loyola University Maryland']

UNITID   OPEID  OPEID6                      INSTNM       CITY STABBR  \
1475  163046  207800    2078  Loyola University Maryland  Baltimore     MD   

             ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
1475  21210-2699           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN  UG12MN   G12MN  \
1475                           NaN       0.1251        0.5534  4175.0  2479.0   

      SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
1475   1095.0                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
1475                       NaN             NaN  

[1 rows x 1986 columns]

**Find and display Johns Hopskins**

In [16]:
df[df['INSTNM']=='Johns Hopkins University']

UNITID   OPEID  OPEID6                    INSTNM       CITY STABBR  \
1473  162928  207700    2077  Johns Hopkins University  Baltimore     MD   

             ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
1473  21218-2688           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN  UG12MN  \
1473                           NaN       0.1316        0.3173  7825.0   

        G12MN  SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
1473  22192.0   1368.0                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
1473                       NaN             NaN  

[1 rows x 1986 columns]

**Calculate the min, max, and average in-state tution of the above four institutions.**

In [20]:
Tuition_cost = df[['INSTNM', 'TUITIONFEE_IN']]

Tuition_cost

INSTNM  TUITIONFEE_IN
0                              Alabama A & M University         9857.0
1                   University of Alabama at Birmingham         8328.0
2                                    Amridge University         6900.0
3                   University of Alabama in Huntsville        10280.0
4                              Alabama State University        11068.0
...                                                 ...            ...
7107                   Arizona State University at Yuma            NaN
7108  Bay Area Medical Academy - San Jose Satellite ...            NaN
7109           High Desert Medical College - Bakerfield            NaN
7110                                       BCI - Malden            NaN
7111               HCI College - Fort Lauderdale Campus        22575.0

[7112 rows x 2 columns]

In [21]:
Tuition_cost[Tuition_cost['INSTNM']=='University of Maryland-College Park']

INSTNM  TUITIONFEE_IN
1479  University of Maryland-College Park        10399.0

In [22]:
Tuition_cost[Tuition_cost['INSTNM']=='Johns Hopkins University']

INSTNM  TUITIONFEE_IN
1473  Johns Hopkins University        52170.0

In [23]:
Tuition_cost[Tuition_cost['INSTNM']=='Loyola University Maryland']

INSTNM  TUITIONFEE_IN
1475  Loyola University Maryland        47560.0

In [24]:
Tuition_cost[Tuition_cost['INSTNM']=='University of Maryland-Baltimore County']

INSTNM  TUITIONFEE_IN
1478  University of Maryland-Baltimore County        11518.0

In [25]:
costs = [10399.0, 52170.0, 47560.0, 11518.0]

costs

[10399.0, 52170.0, 47560.0, 11518.0]

In [26]:
min(costs)

10399.0

In [27]:
max(costs)

52170.0

In [28]:
Tuition_Avg = sum(costs)/ len(costs)

Tuition_Avg

30411.75

**11)Filter the smaller dataset further to only keep colleges from Maryland, Virgina, Washington DC, Delaware, and Pennsylvinia. Save the filtered dataset to a new file named "Five_States_Colleges.csv" for future analysis. Upload this file to your GitHub along with the notebook.**

In [29]:
new_df = df[df['STABBR'].isin(['MD','VA','DC','DE', 'PA'])]

new_df

UNITID    OPEID  OPEID6  \
634     130837  2125200   21252   
635     130873  3025800   30258   
636     130907  1172700   11727   
637     130934   142800    1428   
638     130943   143100    1431   
...        ...      ...     ...   
7063  45794301  4149801   41498   
7084  47503303  4208605   42086   
7089  47503308  4208608   42086   
7094  47503313  4208613   42086   
7103  48305401  4224601   42246   

                                                 INSTNM          CITY STABBR  \
634   Margaret H Rollins School of Nursing at Beebe ...         Lewes     DE   
635                           Dawn Career Institute LLC        Newark     DE   
636          Delaware Technical Community College-Terry         Dover     DE   
637                           Delaware State University         Dover     DE   
638                              University of Delaware        Newark     DE   
...                                                 ...           ...    ...   
7063   Somerset County Technology Center - Bedford Site       Bedford     PA   
7084      Relay Graduate School of Education - Delaware    Wilmington     DE   
7089  Relay Graduate School of Education - Philadelp...  Philadelphia     PA   
7094  Relay Graduate School of Education - Washingto...    Washington     DC   
7103             Barber School of Pittsburgh - Ambridge      Ambridge     PA   

            ZIP  ACCREDAGENCY  INSTURL  NPCURL  ...  \
634       19958           NaN      NaN     NaN  ...   
635       19702           NaN      NaN     NaN  ...   
636       19901           NaN      NaN     NaN  ...   
637       19901           NaN      NaN     NaN  ...   
638       19716           NaN      NaN     NaN  ...   
...         ...           ...      ...     ...  ...   
7063      15522           NaN      NaN     NaN  ...   
7084      19801           NaN      NaN     NaN  ...   
7089  191224906           NaN      NaN     NaN  ...   
7094      20002           NaN      NaN     NaN  ...   
7103  150030000           NaN      NaN     NaN  ...   

      OMENRUP_PARTTIME_POOLED_SUPP  FTFTPCTPELL  FTFTPCTFLOAN   UG12MN  \
634                            NaN          NaN           NaN     53.0   
635                            NaN       0.7108        0.7292    627.0   
636                            NaN       0.5172        0.0923  19385.0   
637                            NaN       0.5579        0.7524   4770.0   
638                            NaN       0.1391        0.5037  20550.0   
...                            ...          ...           ...      ...   
7063                           NaN          NaN           NaN      NaN   
7084                           NaN          NaN           NaN      NaN   
7089                           NaN          NaN           NaN      NaN   
7094                           NaN          NaN           NaN      NaN   
7103                           NaN          NaN           NaN      NaN   

       G12MN  SCUGFFN  POOLYRS_FTFTAIDPCT  FTFTPCTPELL_POOLED_SUPP  \
634      NaN      0.0                 NaN                      NaN   
635      NaN    325.0                 NaN                      NaN   
636      NaN   2092.0                 NaN                      NaN   
637    392.0    941.0                 NaN                      NaN   
638   4305.0   4370.0                 NaN                      NaN   
...      ...      ...                 ...                      ...   
7063     NaN      NaN                 NaN                      NaN   
7084     NaN      NaN                 NaN                      NaN   
7089     NaN      NaN                 NaN                      NaN   
7094     NaN      NaN                 NaN                      NaN   
7103     NaN      NaN                 NaN                      NaN   

      FTFTPCTFLOAN_POOLED_SUPP  SCUGFFN_POOLED  
634                        NaN             NaN  
635                        NaN             NaN  
636                        NaN             NaN  
637                        NaN         

In [30]:
new_df.to_csv(r'Five_States_Colleges.csv', index = False)